### DISCUSSION TITLE
3D version of mxnet.ndarray.UpSampling

### DISCUSSION THREAD

**Discussion by yashvarpatel** [apache/incubator-mxnet#13073] [Q&A]
mxnert.ndarray.UpSampling checks if the data is in form: (batch_size, channels, x,y) (that is, number of dimensions=4), however, I want to do UpSampling for volumetric data where the data is of the form: (batch_size,channels,x,y,z).

Is there a way to do this ?

**Comment by frankfliu** [comment]
@mxnet-label-bot [Question]

**Comment by frankfliu** [comment]
@mxnet-label-bot [NDArray]

**Comment by anirudhacharya** [comment]
currently upsampling only 4-d tensors is supported - https://github.com/apache/incubator-mxnet/blob/master/src/operator/nn/upsampling.cc#L45. 

And from the looks of it you are trying to upsample 5-d tensors. Your best bet for now would be to reshape it to a 4-d tensor or take slices of 4-d array and upsample them separately.

@mxnet-label-bot remove [Question, NDArray] add [Operator, Feature Request]
